.github/workflows/deprecate-version.yml

```
@@ -72,9 +72,9 @@ jobs:
       - name: Deprecate package versions
         env:
           NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+          VERSION: ${{ inputs.version }
```

actionlint (1.7.11)

[error] got unexpected EOF while lexing end marker }}, expecting '}'

(expression)